pom.xml

```diff
@@ -93,11 +93,11 @@
     <commons-lang3.version>3.5</commons-lang3.version>
     <commons-math3.version>3.6.1</commons-math3.version>
     <geogson.version>1.1.97</geogson.version>
-    <google.cloud.bigquery.version>0.32.0-beta</google.cloud.bigquery.version>
+    <google.cloud.bigquery.version>1.36.0</google.cloud.bigquery.version>
     <google.cloud.core.version>1.15.0</google.cloud.core.version>
     <google.findbugs.version>2.0.1</google.findbugs.version>
     <google.http.client.version>1.22.0</google.http.client.version>
-    <google.storage.version>1.7.0</google.storage.version>
+    <google.storage.version>1.23.0</google.storage.version>
```

Comment:
did you test the spanner connection? I'm surprised we don't have to update the spanner library version.

Reply:
yes I did test this for spanner, gcs and bq
![image](https://user-images.githubusercontent.com/14131070/57401953-53183900-718b-11e9-9174-6e1934d3b055.png)